packages/rum-core/src/common/patching/fetch-patch.js

```diff
@@ -106,3 +107,7 @@ export function patchFetch(callback) {
     })
   }
 }
+
+function isAbortError(error) {
+  return error && error.name === 'AbortError'
```

Comment:
All the browsers that we support (the ones from our E2E test) and are compatible with fetch handle this properly. I'm highlighting this, because in MDN 'AbortError' is described as [experimental](https://developer.mozilla.org/en-US/docs/Web/API/DOMException). The legacy ways were: **ABORT_ERR** or code **20**